net/prefix.go

```diff
@@ -3,6 +3,7 @@ package net
 import (
 	"fmt"
 	"math"
+	"net"
```

Comment:
Importing a "net" within "net"? Looks confusing to me. Let's call it "gonet"?

Reply:
valid point